src/org/pahappa/systems/models/Incident.java

```diff
@@ -10,5 +11,56 @@
  *
  */
 public class Incident {
-
+    private int id;
+    private String title;
+    private Type  type;
+    private Status status;
+    private Date  createdOn;
+    private String comment;
+	public int getId() {
+		return id;
+	}
+	public void setId(int id) {
+		this.id = id;
+	}
+	public String getTitle() {
+		return title;
+	}
+	public void setTitle(String title) {
+		this.title = title;
+	}
+	public Type getType() {
+		return type;
+	}
+	public void setType(Type type) {
+		this.type = type;
+	}
+	public Status getStatus() {
+		return status;
+	}
+	public void setStatus(Status status) {
+		this.status = status;
+	}
+	public Date getCreatedOn() {
+		return createdOn;
+	}
+	public void setCreatedOn(Date createdOn) {
+		this.createdOn = createdOn;
+	}
+	public String getComment() {
+		return comment;
+	}
+	public void setComment(String comment) {
+		this.comment = comment;
+	}
+	@Override
+	public String toString() {
+		return "Incident [id=" + id + ", title=" + title + ", type=" + type + ", status=" + status + ", createdOn="
+				+ createdOn + ", comment=" + comment + ", getId()=" + getId() + ", getTitle()=" + getTitle()
+				+ ", getType()=" + getType() + ", getStatus()=" + getStatus() + ", getCreatedOn()=" + getCreatedOn()
+				+ ", getComment()=" + getComment() + ", getClass()=" + getClass() + ", hashCode()=" + hashCode()
+				+ ", toString()=" + super.toString() + "]";
```

Comment:
The toString() method returns the String representation of the object. 
It wouldn't look nice if our Incident object is going to be represented by all this. I think you should use the title to represent the Incident object